### PR TITLE
fix issue #6798

### PR DESCRIPTION
**Pull request by GuoxinYin**
add alias -h for --help

### CODE REVIEW

**Review by marler8997 — APPROVED** [review]

**Comment by GuoxinYin** [comment]
Thank you for approving this! I am new to open source. Could I know what’s the next step? Should I just wait for someone to merge this? And when could we close the original issue? Thank you!

**Comment by marler8997** [comment]
@GuoxinYin yes, you'll have to wait for someone with merge rights to approve and merge.  After it's merged is when you can close the original issue.

**Pull request merged**
